sensors/sensors_test.go

```diff
@@ -19,3 +22,8 @@ func TestTemperatureStat_String(t *testing.T) {
 		t.Errorf("TemperatureStat string is invalid, %v", fmt.Sprintf("%v", v))
 	}
 }
+
+func TestTemperatures(t *testing.T) {
+	// make sure it does not segfault
+	sensors.TemperaturesWithContext(context.TODO())
+}
```

Comment:
`SensorsTemperatures` is defined and it does not require context in argument. And also not just a seg fault, check the return value might be better.

```suggestion
func skipIfNotImplementedErr(t *testing.T, err error) {
        if errors.Is(err, common.ErrNotImplementedError) {
                t.Skip("not implemented")
        }
}

func TestTemperatures(t *testing.T) {
        if os.Getenv("CI") != "" {
                t.Skip("Skip CI")
        }

	v, err := SensorsTemperatures()
	skipIfNotImplementedErr(t, err)
	if err != nil {	
		t.Errorf("error %v", err)
        }
        if len(v) == 0 {	
		t.Errorf("Could not get temperature %v", v)
        }
        t.Log(v)
}
```